### PR TITLE
Implement idea page 

### LINKED ISSUE #11: Project Idea Index Page
- [x] The ideas are displayed in a table with the columns: (id, # reviews, avg rating, title, details, first and last name of the submitter, email and remove)

- [x] The remove button removes a project idea from the list

- [x] The remove button removes the ratings for a given project idea

- [x] The # reviews column should be sortable

- [ ] (Suggestion) Add in a search bar with react-bootstrap-table

### PR DESCRIPTION
**Pull request by btk5h**
Closes #11.

This PR creates an admin-accessible page to the web app which displays the students' ideas in a tabular format. The table allows admins to delete ideas as well as sort them by the number of reviews.

Note: this PR does not add a search bar because of an issue that resides within `react-bootstrap-table-next`. Good luck to whoever wants to take that on in the future.

### CODE REVIEW

**Comment by pconrad** [comment]
@btk5h ... thanks for this.

One question: I had the notion that now.sh might implement an equivalent of Heroku's "review applications", i.e. a deployment specific to the PR.   Is that correct, and if so, what's the URL for that?

I'm asking because it would be helpful to pilot a process that the CS48 teams can use to do their own QA / PR process.     Related question: is there a way on now.sh, similar to Heroku, for one team member to set up deployments for QA / Production for the team, and then add collaborators?

**Comment by btk5h** [comment]
We definitely need some more documentation on how to set up the project in general, but I think that's outside of the scope of this PR.

In particular, the MongoDB instructions are most likely going to end up being quite long, enough that I think it justifies a separate PR.

**Comment by pconrad** [comment]
Though it may be outside the scope of this PR, it isn't possible to test these changes without documentation of how to set up a particular user as an initial admin user.  Whether that's through a configuration file, or through an action that hacks that user into the admin table, it needs to be documented before this PR can go forward.

File: pages/ideas.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    {
+      dataField: "df3",
+      isDummyField: true,
+      text: "delete",
+      formatter: actionsFn,
+    },
+  ];
+}
+
+export default function ManageAdminsPage(props) {
```
Comment (pconrad) [inline pages/ideas.js:104]: @btk5h Should this be "ManageIdeasPage" ?

Comment (pconrad) [reply pages/ideas.js:104]: Came across this when using the code as a model for some other code...

Comment (btk5h) [reply pages/ideas.js:104]: Ah yes! We copied most of the code from the existing admins page, but it looks like we forgot to rename the component.

Comment (andrewhlu) [reply pages/ideas.js:104]: PR #35

**Pull request merged**